tests/fixtures/input-plugins/input-plugin-checkbox.js

```diff
@@ -33,5 +33,5 @@ module.exports = {
       },
     },
   },
-  html: `<label for="{{checkboxSelector.id}}">{{checkboxSelector.label}}</label>{% for option in checkboxSelector.options %}<label><input type="{{checkboxSelector.type}}" name={{checkboxSelector.name}} id="{{checkboxSelector.id}}" value="{{checkboxSelector.value}}" {% if option.value == checkboxSelector.value %}checked{% endif %}>{{option.label}}</label>{% endfor %}`,
-};
\ No newline at end of file
+  html: `{% for option in checkboxSelector.options %}<label for="{{checkboxSelector.id}}--{{option.index}}"><input type="{{checkboxSelector.type}}" name="{{checkboxSelector.name}}" id="{{checkboxSelector.id}}--{{option.index}}" value="{{option.value}}" {% if option.value == checkboxSelector.value %}checked{% endif %}>{{option.label}}</label>{% endfor %}`,
```

Comment:
@Snugug I think we'll end up deep in weeds with this and we should avoid adding caveats for checking if there are checkboxes and if so, if there are multiple checkboxes. That would be straightforward for a simple use case like this, but I expect any further multi-checkbox plugins will have vastly different structure. 
I'm going to add a fieldset around these checkboxes in this fixture, but not add new functionality to sniff for multiple checkboxes to the [renderer](https://github.com/punchcard-cms/content-types/blob/5550f489f113af109d18b159f3dcb7102b79c830/lib/form/html.js#L113).